README.md

```diff
@@ -425,11 +432,16 @@ You _must_ refer to your development board's documentation to determine where yo
 
 ## Telemetry
 
-Currently, the only telemetry that is supported is GPS data.
+Currently, the only telemetry that is supported is battery sensor and GPS data.
 This is because I am still working on the telemetry side of things.
```

Comment:
I can get rid of this entire paragraph, because as soon as I am done with battery telemetry, I am working on adding the rest of the CRSF Protocol's telemetry.

Over time, I can add what telemetry is supported, and folks can keep an eye on my Pull Requests to see what I am adding in next, as opposed to outright saying it here.